### PR TITLE
https://issues.redhat.com/browse/ACM-8962

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-8962

### CODE REVIEW

File: governance/cert_policy_ctrl.adoc [hunk 1]
```diff
@@ -21,6 +21,8 @@ The certificate policy controller is created on your managed cluster. The contro
 
 Certificate policy controller does not support the `enforce` feature.
 
+*Note:* The certificate policy controller expects to find a certificate in a secret in only the `tls.crt` key. If a secret is stored under a different key, add a label named `certificate_key_name` with a value set to the key to let the certificate policy controller know to look in a different key. For example, if a secret contains a certificate stored in the key named `sensor-cert.pem`, add the following label to the secret: `certificate_key_name: sensor-cert.pem`.
```
Comment (jc-berger) [inline governance/cert_policy_ctrl.adoc]: Is "expects to find" the most accurate verb choice? 

If the certificate policy controller _automatically_ looks for a certificate in a secret in the `tls.crt` then I think we could be more direct in our word choice. Something like:

The certificate policy controller automatically looks for a certificate in a secret in only the `tls.crt` key.

**Review by jc-berger — APPROVED** [review]
/lgtm

File: governance/cert_policy_ctrl.adoc (hunk 1 above)
Comment (gparvin) [reply governance/cert_policy_ctrl.adoc]: Thanks, I agree with the update recommended  here.

**Comment by gparvin** [comment]
This looks good once the suggestion provided is handled.  Thanks @dockerymick

File: governance/cert_policy_ctrl.adoc (hunk 1 above)
Comment (dockerymick) [inline governance/cert_policy_ctrl.adoc]: Update after peer review
```suggestion
*Note:* The certificate policy controller automatically looks for a certificate in a secret in only the `tls.crt` key. If a secret is stored under a different key, add a label named `certificate_key_name` with a value set to the key to let the certificate policy controller know to look in a different key. For example, if a secret contains a certificate stored in the key named `sensor-cert.pem`, add the following label to the secret: `certificate_key_name: sensor-cert.pem`. 
```

**Comment by dockerymick** [comment]
Thanks for the reviews, feedback, and approvals! Merging

**Pull request merged**